marda_extractors_api/__init__.py

```diff
@@ -207,6 +241,32 @@ def _parse_python_arg(arg: str):
 
         return function_tree, args, kwargs
 
+    def _execute_python_venv(self, entry_command: str, setup: str):
+        data = None
+
+        with multiprocessing.managers.SharedMemoryManager() as shmm:
+            shm = shmm.SharedMemory(size=1024 * 1024 * 1024)
+            py_cmd = (
+                r'print("Launching extractor"); import '
+                + setup
+                + "; import pickle; import multiprocessing.shared_memory; shm = multiprocessing.shared_memory.SharedMemory(name="
+                + r"'"
+                + r"{}".format(shm.name)
+                + r"'"
+                + "); data = pickle.dumps("
+                + r"{}".format(entry_command)
+                + '); shm.buf[:len(data)] = data; print("Done!")'
```

Comment:
Can we use `f"{}"` notation here instead of `.format()`? It would let us have one command per line too, which might be more legible.

Reply:
this is of course how i tried to do it the first time, but there were all manner of encoding issues, I can give it another go though